content/articles/Introduction-to-Containers-and-Docker/index.md

```diff
@@ -0,0 +1,87 @@
+To comprehend what `Docker` is and why it is used, it is necessary to first understand what `containers` are and what problems they solve. Containers are completely isolated environments built on top of an existing operating system to provide a `virtual barrier` between the app running inside and the rest of the world. Docker, on the other hand, is a piece of software that allows us to manage the lifetime of these containers.
+
+### Table of content
+ 
+- [Container terminologies](#container-terminologies)
+
+- [Introduction to docker/docker engine](#introduction-to-docker-/-docker-engine) 
+ 
+- [Reasons why we use dockers](#reasons-why-we-use-dockers)
+
+- [Docker Windows framework necessities and establishments](docker-windows-framework-necessities-and-establishments) 
+
+- [Conclusion](#conclusion) 
+
+### Container terminologies
+
+1. **Container**: A container is a Container Image that is installed during runtime. It's constructed on top of the Base machine. 
+2. **Container image**: A container image is a self-contained piece of software that contains all of the code, tools, and resources needed to execute.
+3. **Docker file**: It is a book record that contains directions for building a Docker picture.
+4. **Docker Hub**: It is a cloud-based registry that allows you to download Docker images created by other communities.
+5. **Repository**: A Docker repository is a collection of Docker images that all have the same name but differ in tags or versions.
+6. **Registry**: The library is assistance liable for facilitating and circulating pictures.
+7. **Docker Engine**: Docker Engine, is a piece of software that allows us to manage the lifecycle of containers, describe how they will be set up, what applications/software/services they will run within them, their networking and storage needs, and if necessary, how to quickly destroy and restart them.
+8. **Docker Compose**: Docker Compose is an order line instrument for building and running applications made out of numerous Docker containers.
+9. **Orchestration and cluster management**: Orchestration is a tool that monitors and manages the system. In huge compartment arrangements, tooling is crucial for screen and oversee pictures, containers, and hosts through a Command line interface (CLI) or a Graphical user interface(GUI). 
+10. **Volumes**:  Volumes are documents that are mounted on the host and can be imported to different compartments. In addition, it is where all progressions are made to the host filesystem.
```

Comment:
How you've defined the following terminologies is not clear:

1. Container
2. Docker image: ....did you mean, "...building a Docker image?"
3. Registry
4. Docker Compose: What do you mean by order line instrument?
5. Volumes: What do you mean by progressions?

These sentences are either not clear or difficult to follow. Put yourself in the context of a beginner whose reading your article. Either re-write or rephrase the sentences.